src/generateSdk/utils/compileSdk.ts

```diff
@@ -37,11 +37,14 @@ export async function compileSdk(
     language: Language,
     publish: boolean,
     outDir = "../genezio-sdk",
+    ShouldDeleteFolder = true,
```

Comment:
I would rename this parameter to `overwriteIfExists`.


```suggestion
    overwriteIfExists = true,
```